Makefile.inc1

```diff
@@ -2089,9 +2079,10 @@ real-update-packages: stage-packages .PHONY
 	    echo "==> New package $${newpkgname}" ; \
 	  fi ; \
 	done
+.else
+	@echo "==> Bootstrapping repository, not checking for new packages"
 .endif
-	${_+_}@cd ${.CURDIR}; \
```

Comment:
Using make -C here I think could be its own commit.

Reply:
Also, I had this question on the `freebsd-dev` IRC channel at some point, but I don't remember getting a definitive answer on it. Why is the `${_+_}` prefix scattered around everywhere? As I understand, the manpage states that the `+` prefix can be used to force a command to always run, even when `-n` is used at the make command line. So why does it need to be expanded as a variable surrounded by `_` characters?

Reply:
I was curious. It's a local oddity to make the `-N` option not recurse. See 56585ab576ff9265a9c0f44b6d4986eb4f5461e3 and it's predecessor acabf29a1bdafe532c71c1f1a8053c7e32251649